BurnOutSharp/ProtectionType/BD+.cs

```diff
@@ -17,11 +16,56 @@ public string CheckPath(string path, bool isDirectory, IEnumerable<string> files
                 if (files.Any(f => f.Contains(Path.Combine("BDSVM", "00000.svm")))
                     && files.Any(f => f.Contains(Path.Combine("BDSVM", "BACKUP", "00000.svm"))))
                 {
-                    return "BD+";
+                    string file = files.FirstOrDefault(f => Path.GetFileName(f).Equals("00000.svm", StringComparison.OrdinalIgnoreCase));
+                    string version = GetVersion(file);
+                    return (version == null ? "BD+ (Unknown Version)" : $"BD+ Version {version}");
+                }
+            }
+            else
+            {
+                string filename = Path.GetFileName(path);
+                if (filename.Equals("00000.svm", StringComparison.OrdinalIgnoreCase))
+                {
+                    string version = GetVersion(path);
+                    return (version == null ? "BD+ (Unknown Version)" : $"BD+ Version {version}");
                 }
             }
             return null;
         }
+
+        private static string GetVersion(string path)
+        {
+            // Version detection logic from libbdplus was used to implement this
+            try
+            {
+                using (var fs = File.OpenRead(path))
+                {
+                    fs.Seek(0x0d, SeekOrigin.Begin);
+                    int byte1 = fs.ReadByte();
```

Comment:
You went from too verbose to not enough, lol. How about `year1` and `year2` instead?